### ISSUE TITLE
Abnormal sperm structure

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9826]
**Preferred term label:**
Abnormal sperm structure

**Synonyms**

**Definition (free text, please give PubMed ID)**
This term summarises the terms "Abnormal sperm motility (Asthenozoospermia)" and "Abnormal sperm morphology (Teratozoospermia)". 

**Parent term (use hpo.jax.org/app)**
HP:0012041

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 98048, ORPHA: 399980

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
see comment on https://github.com/obophenotype/human-phenotype-ontology/issues/9830
